core/state.go

```diff
@@ -40,6 +41,7 @@ type StateReader interface {
 	ContractNonce(addr *felt.Felt) (*felt.Felt, error)
 	ContractStorage(addr, key *felt.Felt) (*felt.Felt, error)
 	Class(classHash *felt.Felt) (*DeclaredClass, error)
+	CompiledClassHash(classHash *felt.Felt) (*felt.Felt, error)
```

Comment:
I don't think we need this ATM. `blockifier` never makes this call.

https://github.com/NethermindEth/juno/blob/66d5fa523321f4e60a10d4bdbe2668c77cdc6758/vm/rust/src/juno_state_reader.rs#L159-L165